arrayAnalysis.js

```diff
@@ -0,0 +1,18 @@
+const averagefunction = (arrayValue) => {
+  let total = 0;
+  arrayValue.forEach((value) => {
+    total += value;
+  });
+  return total / arrayValue.length;
+};
+
+function analyze(arrayValue) {
+  const maxValue = Math.max(...arrayValue);
+  const minValue = Math.min(...arrayValue);
+  const arrayLength = arrayValue.length;
+  const averageValue = averagefunction(arrayValue);
+  return {
+    average: averageValue, min: minValue, max: maxValue, length: arrayLength,
```

Comment:
- [ ] [OPTIONAL] You could name each method already with the name you wish to return them with and return the object like:
`return {average, min, max, length}`